### ISSUE TITLE
[.NET HTML][Accessibility] [Input.Date Accessibility Issues]

### ISSUE THREAD

**Issue by murcinar** [microsoft/AdaptiveCards#3940]
# Platform
* .NET HTML

# Author or host

Are you an author (like sending something to Outlook), or a host that is rendering your own cards? Host, Finance Digital Assistant

# Version of SDK
4.6.2

What version are you using? Ex: NuGet 1.0.2, or latest master, etc...
Nuget

# Details
We received an accessibility bug on our application where the Input.Date date picker control of the Adaptive Cards is failing to highlight the correct area when expanded using Microsoft Narrator screen reader. Please see the screenshot below:
![image](https://user-images.githubusercontent.com/58231579/79503089-6fe8fd80-7fe5-11ea-9499-502b67a2b731.png)

Also, we were notified the following general accessibility requirements for a date picker as follows:
1.	Ensure the date picker control is accessible with keyboard keys (TAB/Arrow/Enter/Space)
2.	Ensure the controls of date picker have accessible labels to be identified by screen readers 
3.	Ensure the date picker control works with universal calendar shortcut keys 
a.	UP/DOWN Arrow keys moves the focus to same day of previous/next weeks
b.	LEFT/RIGHT Arrow keys moves the focus to previous/next day.
c.	Page Up and Page Down keys moves the focus to same date of previous/next month
d.	Shift + Page Up and Shift + Page Down keys moves the focus to same date of previous/next year
e.	Home key moves the focus to first day of the month
f.	End key moves the focus to last day of the month
g.	Enter/Space key to select the focussed date

Can you please prioritize this to enable accessibility for Input.Date control? 

Thank you!

**Comment by murcinar** [comment]
Hi @shalinijoshi19 , I would appreciate if you can provide us help with this. Please let me know if you have any questions. Thank you!

**Comment by shalinijoshi19** [comment]
@dclaux / @matthidinger thoughts on this one?

**Comment by dclaux** [comment]
There is unfortunately nothing we can do here. The control is the built-in HTML5 date picker, and as per the screenshot it functions normally. My guess is that it's a Narrator "bug" (or quirk, or limitation) where Narrator isn't able to properly compute the screen bounds of the popup.

**Comment by paulcam206** [comment]
As @dclaux mentioned, the output HTML looks something like this:

```html
<input class="ac-input ac-dateInput" name="date" type="date" aria-label="Enter a date in mm/dd/yyyy format" value="2017-10-12" data-ac-separatorid="ac-714fdc6f" style="width: 100%;">
```

The `type="date"` bit combined with it being an `<input>` means that it's up to the browser to render the appropriate control. Indeed, you can see different rendering approaches depending on which browser you use:

### New Edge
![image](https://user-images.githubusercontent.com/16614499/83059326-964f7f00-a00e-11ea-86e8-5502a73942ac.png)
---

### Old Edge 
![image](https://user-images.githubusercontent.com/16614499/83059384-a9fae580-a00e-11ea-85d8-bf576a832761.png)
---

### Firefox
![image](https://user-images.githubusercontent.com/16614499/83059444-c1d26980-a00e-11ea-9712-ea8e67814260.png)
---

### Chrome
![image](https://user-images.githubusercontent.com/16614499/83059536-e9293680-a00e-11ea-9516-c694db60027a.png)
---

Unfortunately, there's not really anything _we_ can do to make this experience more accessible. I'd suggest [filing a bug with your browser vendor](https://support.google.com/chrome/answer/95315).
